src/MSBuildAbstractions/MSBuildHelpers.cs

```diff
@@ -203,10 +203,21 @@ public static bool IsWinForms(IProjectRootElement projectRoot)
         /// </summary>
         public static bool IsDesktop(IProjectRootElement projectRoot)
         {
-            var references = projectRoot.ItemGroups.SelectMany(GetReferences)?.Select(elem => elem.Include);
+            var references = projectRoot.ItemGroups.SelectMany(GetReferences)?.Select(elem => elem.Include.Split(',').First());
             return DesktopFacts.KnownDesktopReferences.Any(reference => references.Contains(reference, StringComparer.OrdinalIgnoreCase));
         }
 
+        /// <summary>
+        /// Determines if a project is a .NET Framework MSTest project by looking at its references.
+        /// </summary>
+        /// <param name="root"></param>
+        /// <returns></returns>
+        public static bool IsNETFrameworkMSTestProject(IProjectRootElement projectRoot)
+        {
+            var references = projectRoot.ItemGroups.SelectMany(GetReferences)?.Select(elem => elem.Include.Split(',').First());
```

Comment:
so the following line will just NRE anyways, no point in getting fancy with `?.`